packages/amplify-e2e-tests/src/__tests__/notifications-fcm.test.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  addNotificationChannel,
+  amplifyStatus,
+  createNewProjectDir,
+  deleteProject,
+  deleteProjectDir,
+  getAppId,
+  initJSProjectWithProfile,
+  removeAllNotificationChannel,
+  removeNotificationChannel,
+} from '@aws-amplify/amplify-e2e-core';
+import {
+  getShortId,
+} from '../import-helpers';
+
+describe('notification category test - FCM', () => {
+  const testChannel = 'FCM';
+  const testChannelSelection = 'FCM  | » Firebase Push Notifications ';
+  const projectPrefix = `notification${testChannel}`.substring(0, 19);
+  const projectSettings = {
+    name: projectPrefix,
+    disableAmplifyAppCreation: false,
+  };
+
+  let projectRoot: string;
+
+  beforeEach(async () => {
+    projectRoot = await createNewProjectDir(projectPrefix);
+  });
+
+  afterEach(async () => {
+    await deleteProject(projectRoot);
+    deleteProjectDir(projectRoot);
+  });
+
+  // Skipping FCM for now until we have setup required e2e accounts
+  it.skip(`should add and remove the ${testChannel} channel correctly when no pinpoint is configured`, async () => {
```

Comment:
Would these tests be better as a [`test.each`](https://jestjs.io/docs/api#testeachtablename-fn-timeout) block? Looks like they are somewhat different so maybe not a good fit, just something to think about.

Reply:
we're not entirely sure how these tests will end up looking after we add the supporting e2e accounts; but will keep an eye on this